packages/calcite-components/src/components/tip-manager/tip-manager.scss

```diff
@@ -1,16 +1,19 @@
+@import "../../assets/styles/header";
+
 /**
  * CSS Custom Properties
  *
  * These properties can be overridden using the component's tag as selector.
  *
- * @prop --calcite-tip-manager-height: The maximum height of the component.
  * @prop --calcite-tip-max-width: The maximum width of a slotted `calcite-tip` within the component.
```

Comment:
Sidebar: we should look into:

1. adding a matching prop on `tip` and deprecate this one or
2. deprecate this prop and let users set max-width/max-inline-size on `tip` directly.